src/transform/commonjs/importCommonjs.js

```diff
@@ -83,7 +83,7 @@ function createImportSpecifier({local, imported}) {
   return new ImportSpecifier({local, imported});
 }
 
-function isVarWithRequireCalls(node) {
+export function isVarWithRequireCalls(node) {
```

Comment:
Extracted in: 075a6bcda920a4668e5620dea2faf8a388599915

Due to dependencies I had to also extract matchRequire and matchRequireWithProperty.
I created two files, but in hindsight I think it would have been reasonable to be in one file.
I can change if you think it helps to keep the file count lower.